sc/src/main/java/uk/ac/ebi/atlas/search/GeneSearchService.java

```diff
@@ -3,10 +3,10 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 import org.springframework.stereotype.Component;
+import uk.ac.ebi.atlas.experimentpage.TsnePlotSettingsService;
 
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.sql.ResultSet;
+import java.util.*;
```

Comment:
IntelliJ tries to be clever here about grouping imports, but it's not very good practice to use `import *`. To fix this, go to IntelliJ IDEA --> Preferences --> Editor --> Java --> Imports tab and change the number in the box (see screenshot below).

![screen shot 2018-10-19 at 10 06 43](https://user-images.githubusercontent.com/5923538/47208968-17508a00-d387-11e8-9895-53a0d9bf85ab.png)